### ISSUE TITLE
Update quickstart example outputs

### ISSUE THREAD

**Issue by gberl002** [openziti/ziti-doc#93]
with the release of 0.25.5 the `ziti edge list...` commands display in a "prettified" view that resembles a table. The quickstart docs (specifically for docker-compose but likely others as well) still reference the older style output and need to be updated to the new output.

Examples should be updated for both `ziti edge list edge-routers` and `ziti edge list identities`

OLD:
```
ziti@724087d30014:/openziti$ ziti edge list edge-routers
id: BZ.Y7vMdAI    name: ziti-edge-router    isOnline: true    role attributes: {}
id: NELWwjMd8    name: ziti-private-blue    isOnline: true    role attributes: {}
id: l9-W7jMf8    name: ziti-fabric-router-br    isOnline: true    role attributes: {}
id: rqZW7vMdA    name: ziti-edge-router-wss    isOnline: true    role attributes: {}
id: xmiYwvMf8    name: ziti-private-red    isOnline: true    role attributes: {}
```

NEW (despite the image, the command `ziti edge list edge-routers` should remain the same in the sample output):
<img width="670" alt="image" src="https://user-images.githubusercontent.com/95235264/166473057-408b270c-5717-493b-83ef-3d5c19845e52.png">

**Disclaimer: Fair warning that the formatting might be tricky converting the console output to look the same in markdown format. I'm not positive it will be cumbersome but it's worth noting as this is tagged "good first issue"**

**Comment by qrkourier** [comment]
Probably easier to use JSON output with `jq` filters or JSONPath if it's really necessary to use the CLI as an API.

**Comment by dovholuknf** [comment]
No. This issue is about updating the doc to align better with what people see now. jq and JSONpath aren't relevant here
